lib/extract-default-messages-transforms/index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = [
+  require("./babel-globalize2-default-into-globalize"),
```

Comment:
For consistency, let's move `lib/extract-default-messages-transforms/babel-globalize2-default-into-globalize.js` to `lib/extract-transforms/`?